Jetcaster/app/src/main/java/com/example/jetcaster/ui/JetcasterAppState.kt

```diff
@@ -58,6 +59,15 @@ class JetcasterAppState(
         isOnline = checkIfOnline()
     }
 
+    fun navigateToPlayer(episodeUri: String) {
+        val encodedUri = Uri.encode(episodeUri)
+        navController.navigate(Screen.Player.createRoute(encodedUri))
```

Comment:
It'd be nice to hook into `isResumed` for the backstack entry to avoid double navigation.

Reply:
Added, thanks!